src/plugins/kibana/public/discover/index.html

```diff
@@ -176,7 +176,12 @@
 
               </header>
 
-              <visualize ng-if="vis && rows.length != 0" vis="vis" es-resp="mergedEsResp" search-source="searchSource"></visualize>
+              <visualize
+                ng-if="vis && rows.length != 0" vis="vis"
+                es-resp="mergedEsResp"
+                export-title="{{opts.savedSearch.title}}"
```

Comment:
I think it makes more sense than the old default of 'table'.
Thoughts @spalger ?